Code of Conduct

```diff
@@ -0,0 +1,22 @@
+Code of Conduct for the Bounty Program
```

Comment:
The formating of the file can be improved by using markDown language:

e.g.
https://github.com/Concordium/workshops-and-hackathons?tab=coc-ov-file https://github.com/Concordium/.github/edit/main/.github/CODE_OF_CONDUCT.md